package.json

```diff
@@ -97,6 +97,7 @@
     "inquirer": "5.2.0",
     "jest": "23.6.0",
     "keyv": "3.1.0",
+    "marked": "0.5.1",
```

Comment:
The `marked` library is now an explicit dependency until such time as a new version of Hexo that bundles this version of the `marked` library is released.